src/main/java/com/tabnine/inline/CompletionAdjustmentType.java

```diff
@@ -0,0 +1,5 @@
+package com.tabnine.inline;
+
+public enum CompletionAdjustmentType {
+  LookAhead
```

Comment:
what's the value of enum with one variant?

Reply:
because of the condition where I check if the snippets should be removed.
I can also add a function that asks if snippets should be removed inside the interface, but I'm not sure it's related to the condition's context. do you have another idea?